server/database/schema.sql

```diff
@@ -1,3 +1,14 @@
+CREATE TABLE user (
```

Comment:
- Missing constraints in this table.
- The field comment_quantity is useless here
- about the is_admin field : What if client ask for a third role tomorrow ?